plugins/dynamic-home-page/docs/layout-options.md

```diff
@@ -0,0 +1,334 @@
+# Layout options
+
+The cards and content of the home page is rendered in a **12-column-grid**.
+
+All cards have a **fixed height** that can be changed by the admin.
+
+![Layout columns](layout-columns.png)
+
+<!--
```

Comment:
does this needs to be commented?

Reply:
It was a long example for the screenshot. But you're right, there are better option to keep that example.